clueless/models.py

```diff
@@ -28,8 +42,9 @@ class Player(models.Model):
     Represents a player in the context of a clueless game.  Ties back to Django user
     """
     user = models.ForeignKey(User)
-    currentSpace = models.ForeignKey(Space)
-    currentGame = models.ForeignKey('Game') # game not defined yet, using string as lazy lookup
+    currentSpace = models.ForeignKey(Space, blank=True)
```

Comment:
Do we actually want this to be null? I feel like it should be set to whatever the starting point is in the game initially (say 0,0 or something)